activation/activation.go

```diff
@@ -862,11 +868,13 @@ func (b *Builder) searchPositioningAtx(
 	if err != nil {
 		return types.EmptyATXID, fmt.Errorf("get latest epoch: %w", err)
 	}
+
 	logger.Info("searching for positioning atx", zap.Uint32("latest_epoch", latestPublished.Uint32()))
+
 	// positioning ATX publish epoch must be lower than the publish epoch of built ATX
 	positioningAtxPublished := min(latestPublished, publish-1)
 	id, err := findFullyValidHighTickAtx(
-		ctx,
+		context.WithValue(ctx, prioritizedVerifyCall, true),
```

Comment:
Yes, but in the end option must be passed to (v *offloadingPostVerifier) Verify() method, which can prioritize jobs.
But it which accepts verifying.OptionFunc, which are defined  in  https://github.com/spacemeshos/post/blob/develop/verifying/verifying_options.go#L8

So it is options of different layer, I would said, and there is no way (at least in current implementation) to read values from options outside package. For me it looks even more strange. 

Another option would be pass just an extra parameter, but we have to set it everywhere.

Reply:
I do not like this option because this is the only place in the whole codebase where adding a value to a context changes behavior of downstream functions. This would also need to be documented on both the actual function that changes behavior and on the parameters that are passed in that context.

Changing the function call - no matter if its a functional option or just an additional parameter - is more straight forward and doesn't require additional documentation (if variable names are self explanatory). It is also against recommendations from the `context` package: https://pkg.go.dev/context#WithValue

> Use context Values only for request-scoped data that transits processes and APIs, _not for passing optional parameters to functions._

Reply:
Fixed using 'option pattern'